lang/cs/Org.Apache.REEF.Tests/Functional/ReefFunctionalTest.cs

```diff
@@ -171,6 +175,21 @@ protected void ValidateSuccessForLocalRuntime(int numberOfContextsToClose, int n
         }
 
         /// <summary>
+        /// Find the number of log lines that contains the message in the log file specified
```

Comment:
We probably don't need empty parameters documentation for protected test-only method.